silkworm/db/state/account_codecs_test.cpp

```diff
@@ -84,28 +84,28 @@ TEST_CASE("AccountSnapshotsCodec") {
         CHECK(codec.encode_word() == *from_hex("010201030000"));
     }
     SECTION("decode") {
-        auto word1 = *from_hex("");
+        auto word1 = BytesOrByteView{*from_hex("")};
```

Comment:
```suggestion
        Word word1{*from_hex("")};
```
(same for word*N* below)